Library/Models/SampleColumnCreate.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nfield.SDK.Models
+{
+    public class SampleColumnCreate
```

Comment:
Adding the action/verb to the model name is a convention that you follow.? It looks bit odd to me not to have just the name `SampleColumn`. Im checking the target API in case that the model naming is part of the API contract

Reply:
Ok found a SampleColumnUpdate in glu API so I guess that there's a contract somewhere.